src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php

```diff
@@ -713,6 +683,15 @@ private function addSessionSection(ArrayNodeDefinition $rootNode): void
                     ->end()
                 ->end()
             ->end()
+            ->validate()
+                ->always(function (array $v): array {
+                    if (!\array_key_exists('handler_id', $v['session'])) {
+                        $v['session']['handler_id'] = isset($v['session']['save_path']) ? 'session.handler.native_file' : null;
```

Comment:
it looks like `save_path` has a default value, so that by default, we'll always have `handler_id` be not null.
we might need to fix the deprecation on 6.4 so that we require save_path to be explicitly set when handler_id is not, so that we can make save_path default to null in 7.0. Can you please have a look?

Reply:
Created the PR for 6.4 here: https://github.com/symfony/symfony/pull/51438

Reply:
Thanks! Time to rebase this PR :)

Reply:
Done :slightly_smiling_face: